test/test.kernel.js

```diff
@@ -138,11 +140,11 @@ describe("Kernel", function () {
         U.withProgram(ctx, squareKern, function (prg) {
           var k = cl.createKernel(prg, "square");
 
-          if (cl.VERSION_1_2) {
-            U.bind(cl.setKernelArg, k, 0, [5, 10, 15])
+          if (cl.v12) {
```

Comment:
more v12 refactors, not sure why

Reply:
I can return back the naming but except for naming itself there are changes related to contents of namespaces, `node-opencl` does not provide deprecated APIs on javascript side, it literally excluded during compile time, I think this is wrong thing to do, if API is there it should be available. As for reasons to change naming is that I find 'v12' more convenient to write, no need to shout `VERSION (press shift) (press minus) 1 (press shift) (press minus) 2`